src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -68,15 +68,19 @@ You can find the config file in your code repo at either /config.json or /demo-c
         }
       },
       "analytics": {
-        "base-currency-code": "USD",
-        "environment": "Production",
-        "store-id": 1,
+        "base-currency-code": "{{YOUR_BASE_CURRENCY_CODE}}",
```

Comment:
I actually think we should maybe rethink how the data is displayed. I don't like all these placeholders, it feels repetitive. Perhaps we provide an example, then describe them in detail below. 

We also have an opportunity here to make a tool that takes an API url, makes a query for these values, and returns the config object exactly as should be saved in `config.json`, or uploaded to public config.

Reply:
Something like this? 

![IMG_2784](https://github.com/user-attachments/assets/5a0245b6-c113-4f60-9197-17150774da92)